### ISSUE TITLE
MOSSY: Color nodes and links in phylogenetic tree view

### ISSUE THREAD

**Issue by roxycintron** [CDCgov/MicrobeTRACE#606]
**Describe the bug**
Nodes and links are not colored when the 'Leaves' and 'Branches' color settings are selected. However, node and link colors display correctly in the 2D network.

**To Reproduce**
1. Load fasta and node files and click Launch
2. Select 'Phylogenetic Tree' view
3. Click the _Settings_ wheel and select the 'Leaves' tab
4. Select 'Show Colors' (the _Global Settings_ open)
5. Select to 'Color Nodes By' _Virus ID_
6. Select to 'Color Links By' _Distance_
7. See error: nodes and links are not colored in the phylogenetic tree view

![image](https://user-images.githubusercontent.com/71835013/200645817-c7d9c155-371a-45bb-9265-b44887b171c6.png)

**Node and link colors are working in 2D network**
![image](https://user-images.githubusercontent.com/71835013/200646295-9e180e25-7753-4a97-85c8-e8a3000e6437.png)

**Desktop (please complete the following information):**
 - Laptop (Windows 10)
 - Browser [Chrome]

**Comment by reagank** [comment]
This may actually be a shortcoming in the tree library, I'm not sure that it allows coloring nodes differently by variable. I don't think this is possible in the current version of MT, either. I'll remove the option for it from the settings UI.

**Comment by roxycintron** [comment]
Yes, nodes are colored in the 2D network and phylogenetic tree in classic
MT. I don't think links are colored in trees though.

**Comment by roxycintron** [comment]
Tested for Mossy today. Nodes are colored if a variable is selected, but links are not colored in the tree if a variable is selected. Closing the issue because I don't think most users change link colors in the tree.

**Comment by billswitzer2** [comment]
Please open this issue again. Thanks!

**Comment by billswitzer2** [comment]
Thanks!

**Comment by roxycintron** [comment]
@billswitzer2  The only issue remaining is that links aren't colored in the tree. The option for color branches is available but not working at this time. Should we "close as not planned"?

**Comment by billswitzer2** [comment]
@reagank Is this an easy fix?

**Comment by billswitzer2** [comment]
I didn’t implement this – it wasn’t possible in prod and it’s not really possible to color the branches differently the way we do with links in the twoD network, so I would advocate for closing it as not planned and I can just take out the reference to branch color in the settings.

From: Switzer, Bill (CDC/NCHHSTP/DHP) ***@***.***>
Sent: Wednesday, May 7, 2025 1:56 PM
To: CDCgov/MicrobeTrace ***@***.***>; CDCgov/MicrobeTrace ***@***.***>
Cc: Mention ***@***.***>; Moscoso, Evan (CDC/NCHHSTP/OD) (CTR) ***@***.***>; Kelly, Reagan (CDC/NCHHSTP/OD) (CTR) ***@***.***>; Cowan, David (CDC/NCHHSTP/DHP) ***@***.***>
Subject: RE: [CDCgov/MicrobeTrace] ANGULAR: Color nodes and links in phylogenetic tree view (Issue #606)

Is this an easy fix?

**Comment by billswitzer2** [comment]
Sounds good. Thanks!

From: Kelly, Reagan (CDC/NCHHSTP/OD) (CTR) ***@***.***>
Sent: Wednesday, May 7, 2025 2:04 PM
To: Switzer, Bill (CDC/NCHHSTP/DHP) ***@***.***>; CDCgov/MicrobeTrace ***@***.***>; CDCgov/MicrobeTrace ***@***.***>
Cc: Mention ***@***.***>; Moscoso, Evan (CDC/NCHHSTP/OD) (CTR) ***@***.***>; Cowan, David (CDC/NCHHSTP/DHP) ***@***.***>
Subject: RE: [CDCgov/MicrobeTrace] ANGULAR: Color nodes and links in phylogenetic tree view (Issue #606)

I didn’t implement this – it wasn’t possible in prod and it’s not really possible to color the branches differently the way we do with links in the twoD network, so I would advocate for closing it as not planned and I can just take out the reference to branch color in the settings.

From: Switzer, Bill (CDC/NCHHSTP/DHP) ***@***.******@***.***>>
Sent: Wednesday, May 7, 2025 1:56 PM
To: CDCgov/MicrobeTrace ***@***.******@***.***>>; CDCgov/MicrobeTrace ***@***.******@***.***>>
Cc: Mention ***@***.******@***.***>>; Moscoso, Evan (CDC/NCHHSTP/OD) (CTR) ***@***.******@***.***>>; Kelly, Reagan (CDC/NCHHSTP/OD) (CTR) ***@***.******@***.***>>; Cowan, David (CDC/NCHHSTP/DHP) ***@***.******@***.***>>
Subject: RE: [CDCgov/MicrobeTrace] ANGULAR: Color nodes and links in phylogenetic tree view (Issue #606)

Is this an easy fix?
